client/common/cmdline.h

```diff
@@ -166,11 +166,20 @@ static const COMMAND_LINE_ARGUMENT_A global_cmd_args[] = {
 	{ "from-stdin", COMMAND_LINE_VALUE_OPTIONAL, "force", NULL, NULL, -1, NULL,
 	  "Read credentials from stdin. With <force> the prompt is done before connection, otherwise "
 	  "on server request." },
+	{ "gateway", COMMAND_LINE_VALUE_REQUIRED,
+	  "g:<gateway>[:<port>],u:<user>,d:<domain>,p:<password>,auto-consent:[on|off],usage-method:["
+	  "direct|detect],access-token:<"
+	  "token>,type:[rpc|http[,no-websockets][,extauth-sspi-ntlm]|auto[,no-websockets][,extauth-"
+	  "sspi-ntlm]],",
```

Comment:
`extauth-sspi-ntlm` option is duplicated, isn't it ?

Reply:
hum in fact the help message seems to have something wrong (if it's right it is so complicated to interpret that it makes it useless)

Reply:
@hardening once for `http` and once for `auto` type